setup.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+### Shell script to spin up a docker container for oracle.
+
+## color codes
+RED='\033[1;31m'
+GREEN='\033[1;32m'
+YELLOW='\033[1;33m'
+CYAN='\033[1;36m'
+PLAIN='\033[0m'
+
+## variables
+ORACLE_CONTAINER="oracle_c"
```

Comment:
Thank you. I think for now the image we use is sufficient, but in the future we can look into it.